jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -443,7 +443,11 @@ def _copy_axes_to_spectral():
                 continue
 
             if isinstance(lyr.layer, GroupedSubset):
-                if not isinstance(lyr.layer.subset_state, RoiSubsetState):
+                subset_state = getattr(lyr.layer, 'subset_state', None)
+                if subset_state is None:
+                    return False
```

Comment:
when would this be `None`?  Is this just intending to skip this layer (if so, I think we want `continue` not `return`)?

```suggestion
                if subset_state is None:
                    continue
```

Reply:
Good call, thanks!